contributing/start.rst

```diff
@@ -56,5 +99,135 @@ Thanks! ❤ ❤ ❤
 
   Zammad Team
 
+.. _translation:
+
+Translation
+============
+
+If you want to contribute to the translations and improve the multi-language
+support of Zammad and/or the documentation, you are welcome to contribute
+as well! The translation of Zammad itself and the documentation is done by using
+Weblate, which is a service for the collaborative translation of projects.
+
+You just have to head over to Zammad in Weblate_. You can either create an
+account (if you don't have one already) or even sign in with your Github
+account!
+
+We will cover some basic steps in the following sections to get started with
+translating. However, if you want to use some additional features of Weblate
+and want to dive deeper into it, their `translation documentation`_ is a good
+starting point.
+
+Basics
+------
+
+The translation of **Zammad** and the translation of the **documentation**
+are splitted in two projects in Weblate. When you click in the top menu under
+"*Projects > Browse all projects*", you can find the overview of the two projects:
+
+.. figure:: /images/contributing/weblate-overview-docs.png
+  :align: center
+  :scale: 65 %
+  :alt: Screenshot showing translated projects in Weblate and menu
+
+  Screenshot showing translated projects and menu bar of Weblate
+
+Structure of translation projects in Weblate:
+
+ * Documentations
```

Comment:
`Documentations` => `Documentation`

We should change this in Weblate first, `Documentations` is not a word!

@mgruner FYI

Reply:
Can someone of you change it in Weblate? I think I can't do it because of permissions. 

Thank you!

Reply:
Renamed in Weblate.

Reply:
Thank you!!